README.md

```diff
@@ -76,6 +76,15 @@ If the version is not recognized, `kubectl-validate` will attempt to look up
 the schemas for the selected version in the official upstream Kubernetes repository
 on GitHub.
 
+## overlay-schemas
+
+The ```--overlay-schemas``` option enables users to apply JSON patches onto resource schemas before they undergo validation. This feature allows for customizing the behavior of schema validation according to specific requirements.
+
+```sh
+kubectl-validate --overlay-schemas <patch_file>
```

Comment:
I don't think the command is correct, aren't you supposed to specify some files you want to validate?

Reply:
yes we are suppose to provide a file to validate this is exactly why I have written patch file in the < >  so user can specify it by themselves